LuckParser/Models/ParseModels/CombatReplay/Actors/RectangleActor.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace LuckParser.Models.ParseModels
+{
+    public class RectangleActor : Actor
+    {
+        private readonly int _height;
```

Comment:
I know I haven't gone through the fields for combat replay stuff yet, but this is better being a public readonly Height; or public Height {get;}, same for width

Reply:
True, I did it this way for consistency with the other classes, but I'll change it if you plan it for the others as well anyways.